src/coreclr/vm/mono/mono_coreclr.cpp

```diff
@@ -147,9 +153,9 @@ MonoDomain* CreateMonoDomainFromObject(MonoObject *o)
 
 CrstStatic g_add_internal_lock;
 
-static SString* s_AssemblyDir;
-static SString* s_EtcDir;
-static SString* s_AssemblyPaths;
+static SString* s_AssemblyDir = NULL;
```

Comment:
Nit, I believe statics are already initialized to 0/NULL?

Reply:
theoretically it should, but I got uninitialized memory for some reasons, perhaps there is some compiler flag in place

Reply:
reverted